ds000246/sub-0001/meg/sub-0001_task-AEF_run-01_meg.json

```
@@ -23,4 +23,4 @@
 "MaxMovement":0.0009892724421254463,
 "DigitizedLandmarks":true,
 "DigitizedHeadPoints":true,
-"AssociatedEmptyRoom":"/sub-emptyroom/meg/sub-emptyroom_task-noise_run-01_meg.ds"}
\ No newline at end of file
+"AssociatedEmptyRoom":"bids::/sub-emptyroom/meg/sub-emptyroom_task-noise_run-01_meg.ds"}
```

We dropped the initial `/`, right?

```suggestion
"AssociatedEmptyRoom":"bids::sub-emptyroom/meg/sub-emptyroom_task-noise_run-01_meg.ds"}
```

---

good catch, we actually don't talk about leading slashes in the `<relative-path>` component. I have made a suggestion here: https://github.com/bids-standard/bids-specification/pull/918#discussion_r928108876

I think trailing slashes in the `<dataset-name>` component are fine.